src/content/guides/scaffolding.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Scaffolding
+contributors:
+  - evenstensberg
+  - pranshuchittora
+sort: 2
+---
+
+It can be hard to set up a complex webpack configuration for the first time. Writing advanced configurations to optimize performance is quite hard. The `init` feature is designed to support people that want to create their own configuration or initializing projects that other people create.
+
+## Creating a scaffold
+
+Before writing a `webpack-cli` scaffold, think about what you're trying to achieve. Do you want a "general" scaffold that could be used by any project or type of app? Do you want something focused, like a scaffold that writes both your `webpack.config.js` and your framework code? It's also useful to think about the user experience for your scaffold.
+
+`webpack-cli` offers interactive experience to customize the output accordingly. For example asking questions like: "What is your entry point?".
+
+### Writing a scaffold
+
+There are many resources where you can learn how to write a scaffold, you can start from: [Writing a Scaffold](/contribute/writing-a-scaffold)
+
+`webpack-scaffold` is a utility suite for creating scaffolds. It contains functions that could be used to create a scaffold.
+
+### Running a scaffold
+
+A scaffold can be executed using `webpack-cli init`:
+
+```bash
+webpack-cli init <your-scaffold>
+```
+
+#### Running a scaffold locally
+
+When the scaffold package is in your local file system you should point `init` to its path:
+
+```bash
+webpack-cli init path/to/your/scaffold
+```
+
+Or you can create a global module and symlink to the local one:
+
+- Using npm
+
+      	```bash
```

Comment:
Please remove extra spacing before before code block lines for npm and yarn examples